src/lib/viewers/image/ImageViewer.js

```diff
@@ -378,6 +379,28 @@ class ImageViewer extends ImageBaseViewer {
         }
     }
 
+    loadUIReact() {
+        super.loadUIReact();
+        this.renderUI();
+    }
+
+    renderUI() {
+        if (this.zoomControls) {
+            this.zoomControls.setCurrentScale(this.scale);
+        }
+
+        if (this.controls && this.options.useReactControls) {
```

Comment:
Should `this.options.useReactControls` be first?

Reply:
I don't think it makes an objective difference? I just prefer the visual symmetry of the existence check being immediately on top of its invocation below.